src/status_im/multiaccounts/login/core.cljs

```diff
@@ -106,6 +106,8 @@
    (wallet/initialize-tokens custom-tokens)
    (wallet/initialize-favourites favourites)
    (wallet/get-pending-transactions)
+   (when (get-in db [:multiaccount :opensea-enabled?] false)
```

Comment:
same here, better to use collectibles instead of opensea

Reply:
and probably better to move this condition inside get-collectibles function

Reply:
The name `opensea-enabled?` is already merged in the backend :|